src/analysis/retail/deathknight/blood/modules/Abilities.ts

```diff
@@ -280,6 +281,7 @@ class Abilities extends CoreAbilities {
         gcd: {
           base: 1500,
         },
+        enabled: !combatant.hasTalent(TALENTS.RAPID_DECOMPOSITION_TALENT),
```

Comment:
This is because of the object in line 261 that has a different cast efficiency.

Reply:
ahhh, I missed that. Rather than duplicate the object, you could make the cast efficiency conditional on the talent (I think). Something like:


```
castEfficiency: hasTalent(...) ? { ... } : undefined;
```

Reply:
will fix!